### ISSUE TITLE
Option to disable overwriting newer files

### ISSUE THREAD

**Issue by Orffyreus** [siom79/jdrivesync#10]
It would be nice to have an option that disables overwriting newer files with older ones.

**Comment by MeMatty** [comment]
I couldn't agree more. I've just started using jdrivesync, in my naivety i thought it would check modified dates, and now have lost a couple of hours work on a word doc.
Other that this, great utility, thanks for developing!

**Comment by Anacletus** [comment]
I agree, it would be wonderful to have that option, or at least the option to make a copy of local files being updated (as I think there is a similar option for deleted files).

I always had to think carefully if I need to download or upload before start working with google drive files.
Now I have added this to my download script, to make a backup before downloading.  Sadly it copy all the files, not just the updated ones. 

LOCALDIR=$HOME/googledrive/sync
REMOTEDIR=/sync
#bk
BKDIR=$(date +%Y%m%d%H%M%S)
cp -ar "$LOCALDIR" "$LOCALDIR/../bk/$BKDIR" 
#Run
